mono/metadata/object-internals.h

```diff
@@ -2160,6 +2160,8 @@ mono_string_empty_handle (MonoDomain *domain);
 gpointer
 mono_object_get_data (MonoObject *o);
 
+#define mono_get_data_unsafe(handle) ((void*)((guint8*)MONO_HANDLE_RAW (handle) + MONO_ABI_SIZEOF (MonoObject)))
```

Comment:
Why the need to diverge so much from the existing mono_object_get_data? Neither the return type, nor the name is equivalent. How about you try to make it more consistent?